README.md

```diff
@@ -58,3 +67,7 @@ published by the Free Software Foundation, either version 3 of the License, or
 ### Disclaimer
 
 The developers of this application does not have any affiliation with the content providers available.
+
+Kotatsu does not host any content on the app, and has no control over what media is put up or taken down.
+
+It merely scrapes 3rd-party websites that are publicly accessable via any regular web browser.
```

Comment:
Also stolen: https://github.com/LagradOst/CloudStream-3/blob/master/README.md

Reply:
These line's are almost on all the website which post movies. So it's generally written everywhere.